doc/orchestrator.rst

```diff
@@ -63,108 +326,207 @@ on that compute host.
+since our database is single-sharded:
 
 .. code-block:: python
 
-  from smartsim import Experiment
-  exp = Experiment("colo-test", launcher="auto")
+  # Initialize a Client
+  colo_client = Client(cluster=False)
+
+.. note::
+    Since there is only one database launched in the Experiment
+    (the colocated database), specifying a a database address
+    is not required when initializing the client.
+    SmartRedis will handle the connection.
+
+.. note::
+   To create a client connection to the colocated database, the colocated Model must be launched
+   from within the driver script. You must execute the Python driver script, otherwise, there will
+   be no database to connect the client to.
+
+Store Data
+----------
+Next, using the SmartRedis client instance, we create and store a NumPy tensor using
+``Client.put_tensor()``:
+
+.. code-block:: python
+
+    # Create NumPy array
+    array_1 = np.array([1, 2, 3, 4])
+    # Store the NumPy tensor
+    colo_client.put_tensor("tensor_1", array_1)
+
+Retrieve Data
+-------------
+Next, retrieve the tensor using ``Client.get_tensor()``:
+
+.. code-block:: python
 
-  colo_settings = exp.create_run_settings(exe="./some_mpi_app")
+    # Retrieve tensor from driver script
+    value_1 = colo_client.get_tensor("tensor_1")
+    # Log tensor
+    colo_client.log_data(LLInfo, f"The colocated db tensor is: {value_1}")
 
-  colo_model = exp.create_model("colocated_model", colo_settings)
-  colo_model.colocate_db_uds(
-          db_cpus=1,              # cpus given to the database on each node
-          debug=False             # include debug information (will be slower)
-          ifname=network_interface # specify network interface(s) to use (i.e. "ib0" or ["ib0", "lo"])
-  )
-  exp.start(colo_model)
+When the Experiment completes, you can find the following log message in `colo_model.out`::
+    Default@21-48-01:The colocated db tensor is: [1 2 3 4]
 
+The Experiment Driver Script
+============================
+To run the application, specify a Model workload from
+within the workflow (Experiment).
+Defining workflow stages requires the utilization of functions associated
+with the ``Experiment`` object.
+In this example, we instantiate an ``Experiment`` object with the name ``getting-started``.
+We setup the SmartSim ``logger`` to output information from the Experiment.
+
+.. code-block:: python
 
-By default, SmartSim will pin the database to the first _N_ CPUs according to ``db_cpus``. By
-specifying the optional argument ``custom_pinning``, an alternative pinning can be specified
-by sending in a list of CPU ids (e.g [0,2,range(5,8)]). For optimal performance, most users
-will want to also modify the RunSettings for the model to pin their application to cores not
-occupied by the database.
+    import numpy as np
+    from smartredis import Client
+    from smartsim import Experiment
+    from smartsim.log import get_logger
+    import sys
 
-.. warning::
+    exe_ex = sys.executable
```

Comment:
Same comment as before -- I think we should add a comment here to say what is going on.